index.js

```diff
@@ -67,7 +67,10 @@ const createMeasurement = async (req, res, client, getCurrentRound) => {
   validate(measurement, 'indexerResult', { type: 'string', required: false })
   validate(measurement, 'minerId', { type: 'string', required: false })
   validate(measurement, 'providerId', { type: 'string', required: false })
-  validate(measurement, 'stationId', { type: 'string', required: false }) // TODO eventually should be required, optional now for backwards compatibility
+  if ('stationId' in measurement) {
+    validate(measurement, 'stationId', { type: 'string', required: false }) // TODO eventually should be required, optional now for backwards compatibility
+    assert(measurement.stationId.length === 64, 400, 'Invalid Station ID')
```

Comment:
How strict do we want to make this assertion? Should we also check that string contains only hex-encoding chars? (E.g. `/^[0-9A-Z]{64}$/`).

Reply:
Good idea! I think [0-9A-F]